src/config.js

```diff
@@ -298,7 +298,7 @@ export const BENCHMARKS = {
   },
 };
 
-const DEFAULT_SUITES = [
+export const DEFAULT_SUITES = [
```

Comment:
I don't think you need to export it if it's not been used somewhere else.
I can't see you using it anywhere else outside of this file. Correct me if I'm wrong.